package.json

```diff
@@ -56,6 +56,7 @@
     "methods": "^1.1.2",
     "module-details-from-path": "^1.0.3",
     "msgpack-lite": "^0.1.26",
+    "mssql": "^5.1.0",
```

Comment:
Use the version in `versions` instead to avoid pulling `mssql` when it's not needed. This is especially true since it's currently in `dependencies` and not `devDependencies` which means it will be pulled by every user.